rest/change-call-state/example-1/example-1.5.x.cs

```diff
@@ -0,0 +1,22 @@
+// Download the twilio-csharp library from twilio.com/docs/csharp/install
+using System;
+using Twilio;
+using Twilio.Http;
+using Twilio.Rest.Api.V2010.Account;
+
+class Example
+{
+  static void Main(string[] args)
+  {
+    // Find your Account Sid and Auth Token at twilio.com/console
+    const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+    const string authToken = "your_auth_token";
+    TwilioClient.Init(accountSid, authToken);
+
+    var url = new Uri("http://demo.twilio.com/docs/voice.xml");
```

Comment:
No need for the additional extra variables `url` and `method`. You're already using named parameters in `CallResource.Update`.